.travis.yml

```diff
@@ -6,22 +6,23 @@ compiler:
 os:
     - linux
     - osx
-matrix:
-  exclude:
-    - os: osx
-      compiler: gcc
```

Comment:
@hppritcha - I think Howard wrote this.  Is there a reason why this was excluded originally?